### DISCUSSION TITLE
Just complimenting the authors

### DISCUSSION THREAD

**Discussion by AutomationMan808** [danpros/htmly#807] [General]
I copied my HTMLy to the root of my domain. 

Cleared the cache
Downloaded
Uploaded
Changed the INI file

Done! No exporting the database or anything. I love flat file systems and I love HTMLy.

Mahalo Bradahs

Rick

**Comment by MrGreen68** [comment]
I can only agree with that. A big thank you to the developer for this fantastic work!

MrGreen
